src/scripts/main.js

```diff
@@ -1,3 +1,227 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+const validScheme = {
+  name: (value) => value.length > 4,
+  age: (value) => value > 18 || value < 90,
+  salary: (value) => `$${value}`,
+};
+
+document.body.insertAdjacentHTML('beforeend', `
+  <form action="/" method="get" class="new-employee-form">
+    <label>
+      Name:
+      <input
+        name="name"
+        type="text"
+        data-qa="name"
+        minlength="4"
+        required
+      >
+    </label>
+    <label>
+      Position:
+      <input name="position" type="text" data-qa="position" required>
+    </label>
+    <label for="office">
+      Office:
+      <select id="office" data-qa="office" required>
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <label>
+      Age:
+      <input name="age" type="number" data-qa="age" required min="18" max="90">
+    </label>
+    <label>
+      Salary:
+      <input name="salary" type="number" data-qa="salary" required>
+    </label>
+    <button type="submit"> Save to table </button>
+  </form>
+`);
+
+const form = document.forms[0];
+const select = form.querySelector('[data-qa="office"]');
+
+const createRow = (...arg) => {
+  const firstField = form.name.value;
+  const secondField = form.position.value;
+  const thirdField = select.value;
+  const fourthField = form.age.value;
+  const fifthField = parseFloat(form.salary.value);
+  const format = fifthField.toLocaleString('en-US');
+
+  tbody.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${firstField}</td>
+      <td>${secondField}</td>
+      <td>${thirdField}</td>
+      <td>${fourthField}</td>
+      <td>$${format}</td>
+    </tr>
+  `);
+};
+
+function showNotification(type, title, text) {
+  form.insertAdjacentHTML('afterend', `
+    <div class="notification" data-qa="notification">
+      <h1 class="title">${title.toUpperCase()}</h1>
+      <p>${text}</p>
+    </div>
+  `);
+
+  const notification = document.querySelector('.notification');
+
+  if (type === 'success') {
+    notification.classList.add('success');
+  };
+
+  if (type === 'error') {
+    notification.classList.add('error');
+  }
```

Comment:
Couldn't you just add type to the classList? 
```suggestion
    notification.classList.add(type);
```

Reply:
@ivbutsykin @Anastasiia-Svintsova thank you!  I fixed everything in the code. I added "if" for event 
```
form.addEventListener('submit', (e) => {
  e.preventDefault();
  if (form.name.value.trim()) {
    createRow();
    showNotification(
      'success',
      'Successful!',
      'New employee is successfully added to the table'
    );
  }
 e.target.reset();
});
```